Mathlib/RingTheory/Localization/Module.lean

```diff
@@ -198,4 +259,27 @@ def LinearMap.extendScalarsOfIsLocalization (f : M →ₗ[R] N) : M →ₗ[A] N
 @[simp] lemma LinearMap.extendScalarsOfIsLocalization_apply (f : M →ₗ[A] N) :
     f.extendScalarsOfIsLocalization S A = f := rfl
 
-end
+variable {M' : Type*} [AddCommMonoid M'] [Module R M'] [Module A M']
+  [IsScalarTower R A M'] (f : M →ₗ[R] M') [IsLocalizedModule S f]
+
+/-- If `(f : M →ₗ[R] M')` is a localization of modules, then the map
+`(localization S) × M → N, (s, m) ↦ s • f m` is the tensor product (insomuch as it is the universal
+bilinear map).
+In particular, there is an isomorphism between `LocalizedModule S M` and `(Localization S) ⊗[R] M`
+given by `m/s ↦ (1/s) ⊗ₜ m`.
+-/
+theorem IsLocalizedModule.isBaseChange : IsBaseChange A f := by
```

Comment:
I think this should go somewhere else. Maybe `Mathlib/RingTheory/Localization/BaseChange.lean`?

Reply:
As suggested, I moved this result to a new file. I kept the authors of the original file (cf. `LocalizedModule.lean`).